### PR TITLE
Parse string bundle config keys + arrays of rules

### PR DESCRIPTION
**Pull request by jsteiner**

### CODE REVIEW

File: lib/cog_api/fake/bundles.ex [hunk 1]
```diff
@@ -83,11 +84,33 @@ defmodule CogApi.Fake.Bundles do
 
   defp to_atom_keys(map) do
     Enum.reduce(map, %{}, fn ({key, val}, acc) ->
+      if is_map(val) do
```
Comment (drapergeek) [inline lib/cog_api/fake/bundles.ex]: Should this be moved to a `cond`?

File: lib/cog_api/fake/bundles.ex [hunk 2]
```diff
@@ -83,11 +84,33 @@ defmodule CogApi.Fake.Bundles do
...
       else
         Map.put(acc, String.to_atom(key), val)
       end
     end)
   end
+
+  defp parse_commands(commands, endpoint) do
+    commands
+    |> Enum.map(fn ({name, command}) ->
+                      create_rules(command.rules, endpoint)
```
Comment (drapergeek) [inline lib/cog_api/fake/bundles.ex]: What do you think about pulling out a function here to make this more readable?

File: lib/cog_api/fake/bundles.ex (hunk 1 above)
Comment (jsteiner) [inline lib/cog_api/fake/bundles.ex]: I don't think so. The `is_map(val)` part is independent of the `is_atom(key)` part below, so I'm not sure how a `cond` would clean it up.

**Comment by drapergeek** [comment]
LGTM

**Pull request merged**